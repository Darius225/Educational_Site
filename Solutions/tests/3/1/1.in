5
21602 5278 3843 19505 13953 
4
2
0
1
4
