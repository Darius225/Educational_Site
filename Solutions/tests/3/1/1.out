3843
21602
5278
13953
